research/src/pages/focusgroup/focusgroup.explainer.mdx

```diff
@@ -561,21 +564,24 @@ linear focusgroups (and vice-versa) as a best practice. The following example is
 Example 12:
 
 ```html
-<horizontal-menu focusgroup="horizontal wrap">
-  <menu-item tabindex="-1">Action 1</menu-item>
-  <menu-item tabindex="0">Action 2</menu-item>
-  <vertical-menu tabindex="-1" focusgroup="vertical extend">
-    <menu-item tabindex="-1">Sub-Action 3</menu-item>
-    <menu-item tabindex="-1">Sub-Action 4</menu-item>
-  </vertical-menu>
+<horizontal-menu role=menubar focusgroup="horizontal wrap">
+  <menu-item role=menuitem tabindex="-1">Action 1</menu-item>
+  <menu-item role=menuitem tabindex="0">Action 2</menu-item>
+  <menu-item role=menuitem tabindex="-1" aria-haspopup=true aria-expanded=true>
+    Action 3
+    <vertical-menu role=menu tabindex="-1" focusgroup="vertical extend">
+      <menu-item role=menuitem tabindex="-1">Sub-Action 3</menu-item>
+      <menu-item role=menuitem tabindex="-1">Sub-Action 4</menu-item>
+    </vertical-menu>
+  </menu-item>
   <menu-item tabindex="-1">Action 5</menu-item>
 </horizontal-menu>
 ```
 
-When focus is on the `<vertical-menu>` focusgroup item, only a down arrow key will descend into the
+When focus is on the "Action 3" menu item, only a down arrow key will descend into the
 nested focusgroup (not a right arrow key because this is disallowed by the vertical extending
```

Comment:
My mental representation of the focusgroup feature got rusty so please take this comment with a grain of salt: I _think_ that if we are indeed on the menu-item of line 570 and we press the down arrow, nothing would happen.

We would need to be on the root of the focusgroup that extend in the orthogonal direction for a key press in the orthogonal direction to move the focus.

Reply:
In other words, the menu-item at line 570 should be the one having `focusgroup="extend vertical"`

Reply:
I know that held true for the time when we had strict parent->child relationships for focusgroups... but when we changed extend to search up the tree to make a connection, I wonder if that changed? E.g., is that linkage for the extending focusgroup bidirectional to allow a descent even if the descender is not an immediate child?

Reply:
(might have to fix the polyfil for this as well...)